### PR TITLE
Several fixes for minecraft.net

### PR DESCRIPTION
**Pull request by awebeer256**

### CODE REVIEW

**Comment by awebeer256** [comment]
For 82124a9, the background is `white` and not `${white}`, because the same selector is also being inverted.

File: src/config/dynamic-theme-fixes.config [hunk 1]
```diff
@@ -19406,6 +19409,28 @@ ul#nav.nav > li.current > a {
...
+.feature-details img
+.ms-accordion .shade
+.ms-accordion td.platform-data
+.nav-links__tab-icon
+
+CSS
+.ms-accordion td.platform-data {
+    background-color: white !important;
+}
+[style*="background-image: url("][style*="/background-images/bg-wool-white.png"],
```
[the remark this answers is not in the document]
Comment (awebeer256) [reply src/config/dynamic-theme-fixes.config]: Done.

File: src/config/dynamic-theme-fixes.config [hunk 2]
```diff
@@ -3062,6 +3062,18 @@ header .logo
...
 
+checkout.minecraft.net
+
+CSS
+symbol#esrb [fill="#000"] {
+    --darkreader-inline-fill: ${black} !important
+}
+symbol#esrb [fill="#fff"] {
+    --darkreader-inline-fill: ${white} !important
+}
```
Comment (Gusted) [inline src/config/dynamic-theme-fixes.config:3073]:
```suggestion
IGNORE INLINE STYLE
symbol#esrb [fill="#000"]
symbol#esrb [fill="#fff"]
```

**Review by Gusted — APPROVED** [review]
Thanks!

**Pull request merged**